system-design/class-diagram/domain/validator.puml

```diff
@@ -0,0 +1,40 @@
+@startuml
+package validator {
+    package exceptions {
+        enum SeverityLevel {
+            Low,
+            Medium,
+            High,
+            Critical
+        }
+
+        class CustomException {
+            + Severity: SeverityLevel
+            + ResolutionSuggestion: string
+        }
+        CustomException <|-- PatronException
+        CustomException <|-- BookException
+        CustomException --( Exception
```

Comment:
What relation is this?

Reply:
This arrow is used to draw relationships between our system entity and a standard library entity,